README.rst

```diff
@@ -1159,9 +1159,14 @@ Choose from the list of available rules:
 
 * **no_unneeded_final_method** [@Symfony, @PhpCsFixer]
 
-  A ``final`` class must not have ``final`` methods and ``private`` method must
+  A ``final`` class must not have ``final`` methods and ``private`` methods must
```

Comment:
As this should be merged on master, fixing the typo can be extracted from this PR to merge it on 2.15.